vlogsApp/vlogsApp/Controller/RacingScreenViewController.swift

```diff
@@ -44,23 +48,23 @@ class RacingScreenViewController: UIViewController, UIScrollViewDelegate {
             self.collectionView.reloadData()
         })
     }
-        
+    
     func mainScreenConfigUI() {
-            
+        
         title = "Racing"
         
         let navBarAppearance = UINavigationBarAppearance()
         navBarAppearance.titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.black]
```

Comment:
Repetion of code , naBarApperance configuration . Use one of the building blocks of OOP to resolve this issue.